Systems/c172p-engine.xml

```diff
@@ -206,13 +206,13 @@
         </switch>
 
         <fcs_function name="auto-engine-mixture">
-            <function>
-                <sum>
+            <function> <!-- mixture to 1 at sea level in autostart, state autostarts and tutorials, default mixture setting is .95 -->
+                <sum
```

Comment:
Please close the tag.
`<sum>`
Otherwise, it works. Autostart tested up to about 9000 ft (in flight).

Reply:
Wow, good catch. Don't know how I manage to do that.